app/scripts/lib/transaction/containers/enforced-simulations.ts

```diff
@@ -123,7 +123,7 @@ function generateDelegation({
   return delegation;
 }
 
-function generateCalldata({
+export function generateCalldata({
```

Comment:
We should re-use `encodeRedeemDelegations` as it's a shared helper, but we definitely don't want to couple to enforced simulations, safest to define the executions explicitly within our hook.